src/migrations/1603170477620-job.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const db = require('../persistence/db');
+
+module.exports.up = async function (next) {
+  const client = await db.connect();
+  await client.query(`CREATE TABLE IF NOT EXISTS jobs (
+      id uuid PRIMARY KEY,
+      title text,
+      salaryRange text,
+      description text,
+      createAt timestamp DEFAULT CURRENT_TIMESTAMP,
+      tags text[],
+      company text,
+      logoURL text DEFAULT 'https://unsplash.com/photos/g2E2NQ5SWSU'
```

Comment:
Just a small note, I don't think the image link works. It will open the whole unsplash website instead of the image only. You can try it https://images.unsplash.com/photo-1593642632559-0c6d3fc62b89?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80

Reply:
I used the direct link of the photo instead. Thank you.